apps/admin_app/lib/admin_app_web/views/dashboard_view.ex

```diff
@@ -0,0 +1,23 @@
+defmodule AdminAppWeb.DashboardView do
+  use AdminAppWeb, :view
+
+  def get_date(params, option) do
+    today = Date.utc_today() |> Date.to_string()
+
+    case option do
+      :from ->
+        case Map.get(params, "from") do
```

Comment:
Instead of handling this with multiple case statements, I would consider using pattern matching here.